src/components/projects.js

```diff
@@ -25,16 +26,22 @@ const Projects = () => {
       }
     }
   `)
+  const intl = useIntl()
   const projectsData = projectsQuery.contentJson.projects
+
   return (
     <div className="partners-container">
       {projectsData.map((node, index) => (
         <Project
           image={node.image.childImageSharp.fluid}
-          title={node.title}
+          title={intl.formatMessage({ id: `projects.${index}.description` })}
+          // title={node.title}
           primaryPartner={node.primary_partner}
           otherPartners={node.other_partners}
           description={node.description}
+          // description={intl.formatMessage({
```

Comment:
this one should work, i'd uncomment it and delete the description right about it. if you try to declare the same prop twice it probably will throw a fit.